xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -597,7 +597,7 @@ void CGUIDialogVideoInfo::Play(bool resume)
     return;
   }
 
-  CFileItem movie(*m_movieItem->GetVideoInfoTag());
+  CFileItem movie(*m_movieItem);
```

Comment:
this looks potentially icky. it will change from an item pointing to the actual underlying file to an item pointing to a library item. not sure if it has consequences but why is this change needed?